docs/reference/jinja2.md

```diff
@@ -83,6 +83,12 @@ Or resize an image and retrieve the resized image object (rendition) for more be
 <div class="wrapper" style="background-image: url({{ background.url }});"></div>
 ```
 
+When working with SVG images, you can use the `preserve_svg` in filter string to prevent operations that would require rasterizing the SVG. When preserve_svg is set to True and the image is an SVG, operations that would require rasterization (like format conversion) will be automatically filtered out, ensuring SVGs remain as vector graphics. This is especially useful in loops processing both raster images and SVGs.
```

Comment:
Grammar fix:
```suggestion
When working with SVG images, you can use `preserve_svg` in the filter string to prevent operations that would require rasterizing the SVG. When preserve_svg is set to True and the image is an SVG, operations that would require rasterization (like format conversion) will be automatically filtered out, ensuring SVGs remain as vector graphics. This is especially useful in loops processing both raster images and SVGs.
```